### ISSUE TITLE
Investigate some kind of microcontroller to give to delegates

### ISSUE THREAD

**Issue by zeth** [PyconUK/pyconuk-2016-internaldocs#69]
Where did we get the debug duck from? Can we get a debug dragon?

**Comment by hawkz** [comment]
Is came from the merch company we used most years. We could ask if they have alternative models. My only hesitation is that historic delegates then have 2...?

**Comment by hawkz** [comment]
Personally I think giving everyone a pi-zero would be much better, and then run a competition for the pi based innovation during the conference or something.

**Comment by ghickman** [comment]
+1 to less tat in the swag bags, a pi-zero with a _purpose_ gets a huge thumbs up from me!

**Comment by zeth** [comment]
That would be great. Another idea would be a esp8266 device, which I think is the or one of the primary target devices for MicroPython?

**Comment by ghickman** [comment]
Yep, there's even a kickstarter funding the development currently!
